README.md

```diff
@@ -498,6 +498,99 @@ Flag | Notes
 `--https_proxy`      | Needed for package manager or Ubuntu guest and for from github downloaded packages
 `--ftp_proxy`       | Needed for some Perfkit packages
 
+CONFIGURATIONS AND CONFIGURATION OVERRIDES
+==================
+Each benchmark now has an independent configuration which is written in YAML.
+A user config is a way to override this default configuration on a per benchmark
+basis. This allows for much more complex setups than previously possible,
+including running benchmarks across clouds.
+
+A benchmark configuration has a somewhat simple structure. It is essentially
+just a series of nested dictionaries. At the top level, it contains VM groups.
+VM groups are logical groups of homogenous machines. The VM groups hold both a
+"vm_spec" and a "disk_spec" which contain the parameters needed to
```

Comment:
Nit: \`-bracket the YAML keys. If disk spec is optional for diskless benchmarks, could you point that out here?